test/support/startblock.rb

```diff
@@ -12,7 +12,7 @@ def remove_project_directory
   def run_startblock(arguments = nil)
     Dir.chdir(tmp_path) do
       Bundler.with_clean_env do
-        %x(#{startblock_bin} #{APP_NAME} #{arguments})
+        %x(bundle exec #{startblock_bin} #{APP_NAME} #{arguments})
```

Comment:
I have to double check this. Since I believe this will not use the local gem code, but rather the installed version of the gem.